doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -0,0 +1,432 @@
+.. _ncs_release_notes_290-nrf54h20-rc1:
+
+|NCS| v2.9.0-nRF54H20-rc1 Release Notes
+#######################################
```

Comment:
I think the following pages need to be updated with the "v2.9.0-nRF54H20-rc1" version:
1. doc/nrf/dev_model_and_contributions/adding_code.rst
2. doc/nrf/shortcuts.txt - If the toolchain manager is also updated.